server/routes/juror-management/reassign/reassign.controller.js

```diff
@@ -52,6 +52,11 @@
               jurorNumber: req.params['jurorNumber']});
           }
 
+          if (typeof req.session.processLateSummons !== 'undefined') {
+            cancelUrl = req.session.processLateSummons.cancelUrl;
+            delete req.session.processLateSummons;
+          }
```

Comment:
I think this will cause issues if the user refreshes the page. The session property `processLateSummons` will no longer be available so the cancel url will be different.

Could you test if that happens? 🤔

Reply:
You're right. I've amended this now :)